BSPConvert.Lib/Source/EntityConverter.cs

```diff
@@ -403,109 +403,18 @@ private void ConvertPlayerStart(Entity playerStart)
 			playerStart.Name = MOMENTUM_START_ENTITY;
 
 			var targets = GetTargetEntities(playerStart);
-			foreach (var target in targets)
+			if (targets.Any())
 			{
-				switch (target.ClassName)
-				{
-					case "target_give":
-						ConvertPlayerStartTargetGive(playerStart, target);
-						break;
-					case "target_init":
-						ConvertPlayerStartTargetInit(playerStart, target);
-						break;
-				}
-			}
-		}
+				var logicAuto = new Entity();
+				logicAuto.ClassName = "logic_auto";
+				logicAuto["target"] = playerStart["target"];
 
-		private void ConvertPlayerStartTargetGive(Entity playerStart, Entity targetGive)
-		{
-			var targets = GetTargetEntities(targetGive);
-			foreach (var target in targets)
-			{
-				if (target.ClassName.StartsWith("weapon_"))
-				{
-					var count = ConvertWeaponAmmoCount(target.ClassName, target["count"]);
-					var weaponName = GetMomentumWeaponName(target.ClassName);
-					var weapon = CreateTargetGiveWeapon(weaponName, playerStart.Origin, count);
-					sourceEntities.Add(weapon);
-				}
-				else if (target.ClassName.StartsWith("ammo_"))
-				{
-					var count = ConvertAmmoCount(target.ClassName, target["count"]);
-					var ammoName = GetMomentumAmmoName(target.ClassName);
-					var ammo = CreateTargetGiveAmmo(ammoName, playerStart.Origin, count);
-					sourceEntities.Add(ammo);
-				}
-				else if (target.ClassName.StartsWith("item_"))
-				{
-					var itemName = GetMomentumItemName(target.ClassName);
-					var item = CreateTargetGiveItem(itemName, playerStart.Origin, target["count"]);
-					sourceEntities.Add(item);
-				}
+				ConvertEntityTargetsRecursive(logicAuto, logicAuto, "OnMapSpawn", 0, new HashSet<Entity>());
```

Comment:
I think you can just pass `playerStart` as the target instead of assigning the logic_auto's target.

Reply:
yeah that works